src/main/groovy/life/qbic/datamodel/dtos/business/Offer.groovy

```diff
@@ -111,21 +117,38 @@ class Offer {
         double itemsWithOverheadNet
         double itemsWithoutOverheadNet
 
-        Builder(Customer customer, ProjectManager projectManager, String projectTitle, String projectDescription, Affiliation selectedCustomerAffiliation) {
+        /**
+         * @deprecated Replaced with {@link #projectObjective}, since 2.1.0
+         */
+        String projectDescription
+
+        Builder(Customer customer, ProjectManager projectManager, String projectTitle, String projectObjective, Affiliation selectedCustomerAffiliation) {
+            /*
+            Overall offer describing properties
+             */
             this.customer = Objects.requireNonNull(customer, "Customer must not be null")
             this.projectManager = Objects.requireNonNull(projectManager, "Project Manager must not be null")
             this.projectTitle = Objects.requireNonNull(projectTitle, "Project Title must not be null")
-            this.projectDescription = Objects.requireNonNull(projectDescription, "Project Description must not be null")
```

Comment:
You need to set the description to an empty String, otherwise implementations will throw NullPointers :)

Reply:
Can I also set it the same as the projectObjective?